nbs/examples/Automatic_Hyperparameter_Tuning.ipynb

```diff
@@ -181,11 +181,23 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        config = {**nhits_default_config(trial)}
</pre><p>The function already returns a dict, so there's no need to unpack it, you can assign it to the config variable directly or unpack it and return it in a single statement like:</p><pre class="ql-syntax" spellcheck="false">def config_nhits(trial):
    return {
        **nhits_default_config(trial),
        'random_seed': trial_suggest_int(...),
        ...
    }
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/927/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Thanks - agree although that solution implicitly requires users to perform the operation in exactly that order, i.e.

```
return {
    **nhits_default_config(trial),
    'random_seed': trial_suggest_int(...),
    ...
}
```
works, but this:

```
return {
    'random_seed': trial_suggest_int(...),
    **nhits_default_config(trial),
    ...
}
```
will not correctly update the parameters. That's why I thought to make the order and update step explicit.